src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { boolean, text, select } from '@storybook/addon-knobs';
+import { State, Store } from '@sambego/storybook-state';
+import { action } from '@storybook/addon-actions';
```

Comment:
It would be good to add the A11Y addon per component also.

Reply:
I added it globally.
<img width="1168" alt="screen shot 2019-02-13 at 15 17 01" src="https://user-images.githubusercontent.com/12142840/52717912-7ecbbe80-2fa2-11e9-8127-10228dc28e9d.png">

Reply:
I think this means that it will run A11Y tests on all components including the other Addons components?

Reply:
I will add A11Y addon in separate task.